src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -0,0 +1,184 @@
+        );
+      })
+  )
+  .fn(async t => {
+    const { format, usageType, usageCopy } = t.params;
+    const { blockWidth, blockHeight, feature } = kTextureFormatInfo[format];
+    await t.selectDeviceOrSkipTestCase(feature);
+    t.device.destroy();
+    t.expectValidationError(() => {
+      t.device.createTexture({
+        size: { width: blockWidth, height: blockHeight },
+        usage: kTextureUsageTypeInfo[usageType] | kTextureUsageCopyInfo[usageCopy],
+        format,
+      });
+    });
+  });
+
+g.test('createView,2d,uncompressed_format')
+  .desc(
+    `A texture from a destroyed device should not be able to create any valid 2d uncompressed texture views.`
+  )
+  .params(u =>
+    u
+      .combine('format', kRegularTextureFormats)
+      .beginSubcases()
+      .combine('usageType', kTextureUsageType)
+      .combine('usageCopy', kTextureUsageCopy)
+      .filter(({ format, usageType }) => {
+        const info = kTextureFormatInfo[format];
+        return !(
+          (!info.renderable && usageType === 'render') ||
+          (!info.storage && usageType === 'storage')
+        );
+      })
+  )
+  .fn(async t => {
+    const { format, usageType, usageCopy } = t.params;
+    const { blockWidth, blockHeight } = kTextureFormatInfo[format];
+    const texture = t.device.createTexture({
+      size: { width: blockWidth, height: blockHeight },
+      usage: kTextureUsageTypeInfo[usageType] | kTextureUsageCopyInfo[usageCopy],
+      format,
+    });
+
+    t.device.destroy();
+    t.expectValidationError(() => {
+      texture.createView({ format });
+    });
+  });
+
+g.test('createView,2d,compressed_format')
+  .desc(
+    `A texture from a destroyed device should not be able to create any valid 2d compressed texture views.`
+  )
+  .params(u =>
+    u
+      .combine('format', kCompressedTextureFormats)
+      .beginSubcases()
+      .combine('usageType', kTextureUsageType)
+      .combine('usageCopy', kTextureUsageCopy)
+      .filter(({ format, usageType }) => {
+        const info = kTextureFormatInfo[format];
+        return !(
+          (!info.renderable && usageType === 'render') ||
+          (!info.storage && usageType === 'storage')
+        );
+      })
+  )
+  .fn(async t => {
+    const { format, usageType, usageCopy } = t.params;
+    const { blockWidth, blockHeight, feature } = kTextureFormatInfo[format];
+    await t.selectDeviceOrSkipTestCase(feature);
+    const texture = t.device.createTexture({
+      size: { width: blockWidth, height: blockHeight },
+      usage: kTextureUsageTypeInfo[usageType] | kTextureUsageCopyInfo[usageCopy],
+      format,
+    });
+
+    t.device.destroy();
+    t.expectValidationError(() => {
+      texture.createView({ format });
+    });
+  });
+
+g.test('createSampler')
+  .desc(`A destroyed device should not be able to create any valid samplers.`)
+  .fn(async t => {
+    t.device.destroy();
+    t.expectValidationError(() => {
+      t.device.createSampler({});
```

Comment:
I think the descriptor should be optional

```suggestion
      t.device.createSampler();
```

Reply:
Done.